### ISSUE TITLE
OSError: [Errno 20] Not a directory

### ISSUE THREAD

**Issue by bigrig2212** [robotframework/SeleniumLibrary#1132]
Hi. I'm a first time user of Robotframework and running into this error out of the gate with the following script and testing all browsers. Any idea how to fix? 

`*** Settings ***
Documentation  This is some basic info
Library  SeleniumLibrary

*** Variables ***

*** Test Cases ***
User must sign in to check out
	[Documentation]  some info about the test
	[Tags]	Smoke
	Open Browser   http://www.amazon.com  ie
	Sleep  3s

*** Keywords ***`

**Comment by aaltat** [comment]
Would you show how you run the test?

**Comment by aaltat** [comment]
Is this still a problem?

**Comment by bigrig2212** [comment]
Oh hi. Thanks so much for following up.
I gave up on that computer and tried two others - where I got it working
successfully both times (one was the same model as the non-working
computer).

For the sake of debugging, in case you're interested, I went back to the
non-working computer and did a screen recording to show you how I'm running
the tests and the error: https://photos.app.goo.gl/ccivDFIYKw6rlrv33

I can only imagine that it has something to do with conflicting software /
bad installation of some sort. Although I went through the same
installation steps on each.

thanks,
ben
